integration_tests/two_sided_pay_as_clear_market.feature

```diff
@@ -27,7 +27,9 @@ Feature: Two sided pay_as_clear market tests
      When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a for 2 hrs
      Then there are files with offers, bids & market_clearing_rate for every area
 
+  # TODO: re-enable in context of bug/D3ASIM-3534
   @slow
+  @disabled
```

Comment:
Hmm, so now an additional test is failing? Do you have a clue about the reason already ?

Reply:
Ah yes. Thanks for trying the slow tests from time to time!